tests/test_logger.py

```diff
@@ -296,6 +297,19 @@ def test_report_figure_to_unsupported_format_raises_error(tmp_path, unsupported_
     writer.close()
 
 
+@pytest.mark.parametrize("unsupported_format", ["stdout", "log", "json", "csv"])
```

Comment:
is the new feature also tested somewhere?
currently only the failure case is tested?

Reply:
I tested the new feature in practice, but not programmatically. It is not as easy as for the other types of logs (e.g. images) :
> I did not create a test_report_hparam_to_tensorboard test, as hparams logs are not seen by EventAccumulator (c.f. [Stack Overflow](https://stackoverflow.com/questions/70033253/read-out-tensorboard-hparams-programatically) and [EventAccumulator implementation](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/backend/event_processing/event_accumulator.py)).

Two solutions are proposed in [this stackoverflow page](https://stackoverflow.com/questions/70033253/read-out-tensorboard-hparams-programatically) but one requires an external library, and the other is not very clean.

Reply:
more than reading the logged hparams, we should at least run the logger.